apps/docs-snippets/src/utils.ts

```diff
@@ -41,7 +41,7 @@ export const getTestWallet = async (seedQuantities?: CoinQuantityLike[]) => {
   });
 
   // add the UTXO inputs to the transaction request
-  request.addResourceInputsAndOutputs(resources);
+  request.addResources(resources);
```

Comment:
I made this change because I noticed that the original method name `addResourceInputsAndOutputs` does not completely describe the function's behavior. Specifically, it doesn't clarify the type of output being added, which could lead to confusion.

We have two possible types of inputs that are resources: `Coin` and `Message`. However, it is not clear what kind of output is being added. So since it was not completely descriptive, I've gone for a shorter name.

In order to improve readability I've implemented 3 methods: `addCoinInput`, `addMessageInput`, and `addChangeOutput`.  Each one handles specifically what their name suggests. Therefore, the `addResources` method is now just a helper method that calls these more specific methods.

Also, since I decided to add a specific add resources method for predicates, to specifically require the predicate byte and avoid users forgetting to pass it when calling `addResources`, I got under the impression that `addResourceInputsAndOutputsForPredicate` would end up being too long/descriptive.

Maybe I am just nitpicking here. I do not have a strong opinion on this, Let me hear what you guys think, if you think `addResourceInputsAndOutputs` does a better job here, I roll back the change.